ecl/hqlcpp/hqlres.cpp

```diff
@@ -174,13 +175,13 @@ void ResourceManager::addManifestFile(const char *filename, ICodegenContextCallb
     bool isSigned = false;
     const char *useContents = fileContents;
     // Check for signature
-    if (startsWith(fileContents, "-----BEGIN PGP SIGNED MESSAGE-----"))
+    if (queryCodeSigner().hasSignature(fileContents))
     {
         try
         {
-            OwnedHqlExpr sig = checkSignature(fileContents.length(), fileContents.str());
-            useContents = stripSignature(strippedFileContents, fileContents).str();
-            isSigned = true;
+            StringBuffer signer, errmsg;
+            isSigned = queryCodeSigner().verifySignature(fileContents, signer, errmsg);
```

Comment:
no longer throws if there is an error.

Reply:
I've changed verifySignature to throw exceptions instead of return error messages in errmsg.